packages/router/__tests__/router-test.ts

```diff
@@ -3072,12 +3079,7 @@ describe("a router", () => {
           },
         });
       });
-    });
 
-    describe(`
-      A) POST /foo |--|--X
-      B) GET  /bar       |-----O
-    `, () => {
```

Comment:
Had two identical describe blocks here testing behavior for `actionReload` versus `submissionRedirect`, so condensed both tests into one `describe`